gui.go

```diff
@@ -505,6 +508,12 @@ func (g *Gui) MainLoop() error {
 		if err := g.flush(); err != nil {
 			return err
 		}
+		// used during testing for synchronization
+		if g.testNotify != nil && g.testCounter > 0 {
+			for ; g.testCounter > 0; g.testCounter-- {
+				g.testNotify <- struct{}{}
+			}
```

Comment:
I think this looks a bit more GOish
```suggestion
			for g.testCounter > 0 {
				g.testCounter--
				g.testNotify <- struct{}{}
			}
```

Reply:
fixed